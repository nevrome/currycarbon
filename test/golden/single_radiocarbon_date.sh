currycarbon "3000,30"