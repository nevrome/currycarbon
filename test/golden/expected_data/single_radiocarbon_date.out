currycarbon v0.3.0.1 (UTF-8)
Method: Bchron {distribution = StudentTDist {ndf = 100.0}}
Curve: IntCal20
Calibrating...
Done.
CalEXPR: [1] 1:3000±30BP
Calibrated: 1379BC >> 1364BC > 1238BC < 1131BC << 1124BC
1-sigma: 1364-1360BC, 1282-1197BC, 1169-1163BC, 1140-1131BC
2-sigma: 1379-1344BC, 1304-1124BC
                              ▁▁▁▁▁▁                             
                             ▁▒▒▒▒▒▒▁                            
                            ▁▒▒▒▒▒▒▒▒▁                           
               ▁          ▁▁▒▒▒▒▒▒▒▒▒▒▁    ▁   ▁▁                
              ▁▒▁       ▁▁▒▒▒▒▒▒▒▒▒▒▒▒▒▁▁▁▁▒▁  ▒▒                
            ▁▁▒▒▒▁▁    ▁▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▁▁▒▒▁               
        ▁▁▁▁▒▒▒▒▒▒▒▁▁▁▁▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁
 -1410 ┄──┬─────────────┬─────────────┬─────────────┬────────────┄ -1020
             > >                 ^              <<               
               ─          ──────────────   ─   ──                
             ──────    ───────────────────────────               
